### ISSUE TITLE
Continue after error

### ISSUE THREAD

**Issue by dgaleareading** [keras-team/keras-tuner#366]
Hi,

I was performing a search but my code crashed. Would it be possible to continue from where the search stopped?

Thanks.

**Comment by yixingfu** [comment]
yes. set overwrite to false (which is default) and run the same code it should pick up from previous search.
